### ISSUE TITLE
FeatureRequest: Copy Snapshot to clipboard

### ISSUE THREAD

**Issue by WiIson** [geovens/gInk#12]
This is a great tool that I will use instead of zoomit. One addition that would be great is to automatically copy the sanpshot to the clipboard for pasting in other docs/chat/emails.

**Comment by geovens** [comment]
Hi Wilson. The snapshot is copied to the clipboard. Please let me know if you are sure that it doesn't work for you.

**Comment by WiIson** [comment]
Hi, Thanks for the reply. It is definitely not working for me. I am using
windows 10. I use other screenshot capture tools like greenshot and they do
save their captures to the clipboard fine.

Thanks

**Comment by tony-loa** [comment]
Hi, I'm able to screenshot and save it on the designated folder. 
I'm using Win 7 SP1.
Hope it helps

**Comment by geovens** [comment]
> Hi, I'm able to screenshot and save it on the designated folder.
Hi, do you mean the screenshots are saved to image files but not copied to the clipboard?

**Comment by tony-loa** [comment]
Yes, screenshots are saved automatically to the folder and I can paste into any software (MS Word, etc)

Very useful for presentation. Please keep maintaining this little gem. Thank you.

**Comment by WiIson** [comment]
Yes it is a great app and it does save the image to a folder. The feature
request was for the application to instantly copy the image to your
clipboard in addition to saving the image to a folder. It sounds like
that's what the app is supposed to do anyway. There must just be some
conflict in my system I guess. I create a lot of clips and paste into chats
for feedback on peoples work so, for me, I need instant copy to my
clipboard.  I found another app that does this now though. Thanks.

**Comment by geovens** [comment]
Tbh now I have no idea why it doesn't work for you. I will keep note of the issue and hope to find out the cause someday.
